lib/twilio/rtc/rtcpc.ts

```diff
@@ -22,7 +22,7 @@ function RTCPC(options) {
   if (options && options.RTCPeerConnection) {
     this.RTCPeerConnection = options.RTCPeerConnection;
   } else if (util.isLegacyEdge()) {
-    this.RTCPeerConnection = new RTCPeerConnectionShim(typeof window !== 'undefined' ? window : global);
+    this.log.info('This browser is not supported.');
```

Comment:
We should throw an error

Reply:
Actually let's just remove this else if block since it's EOS. Let's not make this complicated.